lightning_examples/distributed-training/main.ipynb

```diff
@@ -0,0 +1,1345 @@
+{
```

Comment:
ipynb is autogenerated

Reply:
I need this first to generate the other file with jupytext, will be removed